functions-and-scope.js

```diff
@@ -71,8 +110,14 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Log het antwoord in de terminal.
 
 // ---- Verwachte uitkomst: 9
-
-
+let highestGrade3a = 0;
```

Comment:
Het is handiger om als default waarde de eerste waarde uit de array te pakken in plaats van `0`. Stel je voor dat je `const grades = [-2, -4, -1];` als waarde hebt, dan zou `highestGrade3a` uiteindelijk de waarde `0` hebben terwijl die waarde helemaal niet eens in de array voorkomt 😉

Reply:
Heel goed punt, dank je wel!